pycti/entities/opencti_case_rfi.py

```diff
@@ -100,6 +100,23 @@ def __init__(self, opencti):
             name
             description
             information_types
+            severity
+            priority
+            tasks {
```

Comment:
I'm wondering a bit about the way we add parameters we need for connectors here in the default properties that we use for `list` and `get` methods. 
Do we have a best practice about what we should add ? or we can add everything ? Can it have an impact on connectors performance ? @richard-julien @Kedae

Reply:
I see no problem doing this in the client-python if we need the info within the connector. What is costy is the API call, not the number of attributes we get.